srcpkgs/aspell-el/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'aspell-el'
+#
+#	This is copy of aspell's site
+#	https://ftp.gnu.org/gnu/aspell/dict/0index.html
+#
+#	Project's page: http://www.elspell.gr/
+#
+#	Void Linux Package:
+#	Nicholas Christopoulos <nereus@freemail.gr>
+#
+pkgname=aspell-el
+version=0.08
+revision=0
```

Comment:
Revision means which time same version of package is built in Void, so distfiles can't depend on it. For avoiding dash in version, do like in libcli template.